ts/receiver/configMessage.ts

```diff
@@ -53,14 +54,6 @@ async function handleGroupsAndContactsFromConfigMessage(
   envelope: EnvelopePlus,
   configMessage: SignalService.ConfigurationMessage
 ) {
-  const didWeHandleAConfigurationMessageAlready =
-    (await getItemById(hasSyncedInitialConfigurationItem))?.value || false;
-  if (didWeHandleAConfigurationMessageAlready) {
-    window?.log?.info(
-      'Dropping configuration contacts/groups change as we already handled one... '
-    );
-    return;
-  }
```

Comment:
you have to keep this check on so we do not run the closed group/opengroup logic everytime.

but you also need to keep to run the handlecontactreceived part, even if this `didWeHandleAConfigurationMessageAlready`  is off.

So I think you could just pu the if back ON,

and 

```
  if (didWeHandleAConfigurationMessageAlready) {
    window?.log?.info(
      'Dropping configuration groups change as we already handled one... Only handling contacts '
    );
if (configMessage.contacts?.length) {
    await Promise.all(configMessage.contacts.map(async c => handleContactReceived(c, envelope)));
}
    return;
  }
```

Also, you need to disable this logic when the featureflag is off